target/linux/ath79/nand/base-files/etc/board.d/02_network

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+
+. /lib/functions/system.sh
+. /lib/functions/uci-defaults.sh
+
+ath79_setup_interfaces()
```

Comment:
Same as for 01_leds? I would have put that in normal base-files...

Reply:
ibid